app.js

```diff
@@ -0,0 +1,112 @@
+const div = document.querySelector('#menu');
+const icon = document.querySelector('#icon');
+icon.addEventListener('click', () => {
+  div.classList.toggle('menu_holder_display');
+});
+
+const data = {
+  project1: {
+    img: './assets/main/projectdetails.jpg',
+    deskimg: './assets/main/port.jpg',
+    head: 'Keeping track of hundreds of components',
+    para: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+  },
```

Comment:
- [ ] Each project card should have at least 5 things **(name, image, description, technologies, live-link)**. Please add them because you are using this template to generate your card :+1: